charts/lagoon-core/Chart.yaml

```diff
@@ -42,3 +42,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: changed
       description: update lagoon appVersion to v2.24.1
```

Comment:
This can be removed, since 2.24.1 has been released now.

The changes in this PR would effectively be the start of the next release.

Reply:
Same for the `lagoon-test` chart yaml.

Reply:
cool - done.